documentation/milestones/individual_milestones/static_site_improvements.md

```diff
@@ -0,0 +1,208 @@
+
+- Which languages should we commit to providing translation support for?
+- Which criteria should determine whether or not content needs to be translated (e.g. audience, source, where it's hosted)?
+- Should we adopt a tool that assists with localization and translation (e.g. [Crowdin](https://crowdin.com/))?
+- How can/should translations be approved and moderated before they are published?
+
+### Feedback Mechanism
+
+As part of this deliverable we should define a system for soliciting user input and feedback via the site. This feedback mechanism should:
+
+- Enable users to provide feedback or ask questions directly from the site
+- Enable internal stakeholders to solicit input from users to specific questions or topics
+- Allow internal stakeholders to view all responses
+
+### Definition of done
+<!-- Required -->
+
+- [ ] Basic requirements:
+  - [ ] Code is deployed to `main` & PROD through our CI/CD pipeline
+  - [ ] Services are live in PROD (may be behind feature flag)
+  - [ ] Translations are live in PROD (or have a defined timeline for completion)
+  - [ ] All new services have passed a security review (if necessary)
+  - [ ] All new services have completed a 508 compliance review (if necessary)
+  - [ ] Data needed for metrics is actively being captured in PROD
+  - [ ] Key architectural decisions made about this deliverable are documented publicly
+- [ ] Functional requirements for content management:
+  - [ ] Internal stakeholders can initiate content changes through a formal process and/or tool that is accessible to non-technical users
+  - [ ] Internal stakeholders can preview content and design changes before they become visible to all users on the PROD version of the site
+  - [ ] A user guide describing the content management process/system has been published in GitBook
+- [ ] Functional requirements for internationalization:
+  - [ ] Static site content is supported in *at least* 3 languages
+  - [ ] Users can change the default language from any page of the static site
+  - [ ] A formal process and/or tool has been adopted to facilitate content translations
+  - [ ] A user guide describing the translation process has been published in GitBook
+- [ ] Functional requirements for feedback mechanism:
+  - [ ] Users can provide feedback or ask questions about Simpler Grants.gov directly from the site
+  - [ ] Internal stakeholders can view visitor feedback and/or questions in a centrally location
+  - [ ] A user guide describing how to access and manage user feedback has been published in GitBook
+
+### Proposed metrics for measuring goals/value/definition of done
+<!-- Required -->
+
+1. Total number of site visits by language
+2. Total number of form responses
+3. Number of unique site visitors by language
+4. Number of unique form respondents
+5. Percentage of site content translated into each supported language
```

Comment:
Does this mean something like, in October, 80% of traffic was in English, 10% in Spanish, and 10% in French?

Reply:
What you're describing is more like `Number of unique site visitors by language`

The `Percentage of site content translated into each supported language` is more like what you see on Open Collective's website (see screenshot below) which is calculated automatically with a tool like [Crowdin](https://crowdin.com/project/opencollective)

<img width="297" alt="Screenshot 2023-11-07 at 1 32 14 PM" src="https://github.com/HHS/simpler-grants-gov/assets/21350331/6c07d98f-50f9-44c2-895f-17a7a79a24a5">